estat/README.md

```diff
@@ -0,0 +1,156 @@
+## Overview
+
+`estat` - extensible performance observability tool for Delphix Dynamic Data Platform.  
+
+This tool mainly uses `eBPF` programs with `BCC` front-end to collect and output performance statistics from kernel via `kprobes`.  
+It is made up of the following components:
+
+- `cmd/estat`
+     - Main script that provides a framework for common I/O data aggregation and drives execution of various *performance data collectors*. 
```

Comment:
I think we should probably wrap lines at some reasonable length (80 or 100 characters)

Reply:
Ok sounds good.